drivers/at/at.c

```diff
@@ -334,118 +390,110 @@ ssize_t at_send_cmd_get_lines(at_dev_t *dev, const char *command,
 
     res = at_send_cmd(dev, command, timeout);
     if (res) {
-        goto out;
+        return res;
     }
 
     memset(resp_buf, '\0', len);
 
-    while (1) {
-        res = at_readline(dev, pos, bytes_left, keep_eol, timeout);
+    bool first_line = true;
+    while (bytes_left) {
+        if (first_line) {
```

Comment:
This function added empty responses to the response list too. We keep that behavior, but we don't want the first empty line in. See the comments in `at_send_cmd()`.

Reply:
Better add those comments to the code, future readers won't have the Github comments at hand.

Reply:
I will change this behavior anyway in a future PR regarding URCs, because it doesn't make sense (the first n lines might be URCs). Additionally, this behavior is peculiar. I don't see why we would care about empty lines, and why the DCE would send them in the first place.